mmv1/third_party/terraform/services/securitycenterv2/resource_scc_v2_organization_notification_config_test.go

```diff
@@ -0,0 +1,81 @@
+package securitycenterv2_test
+
+import (
+    "testing"
+
+    "github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+    "github.com/hashicorp/terraform-provider-google/google/acctest"
+    "github.com/hashicorp/terraform-provider-google/google/envvar"
+)
+
+func TestAccSecurityCenterV2OrganizationNotificationConfig_basic(t *testing.T) {
+    t.Parallel()
+
+    context := map[string]interface{}{
+        "org_id":        envvar.GetTestOrgFromEnv(t),
+        "config_id":     acctest.RandString(t, 10),
+        "random_suffix": acctest.RandString(t, 10),
+    }
+
+    acctest.VcrTest(t, resource.TestCase{
+        PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+        ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+        Steps: []resource.TestStep{
+            {
+                Config: testAccSecurityCenterOrganizationNotificationConfig_basic(context),
+            },
+            {
+                ResourceName:      "google_scc_v2_organization_notification_config.default",
+                ImportState:       true,
+                ImportStateVerify: true,
+                ImportStateVerifyIgnore: []string{
+                    "parent",
+                    "config_id",
+                },
+            },
+            {
+                Config: testAccSecurityCenterOrganizationNotificationConfig_update(context),
+            },
+        },
+    })
+}
+
+func testAccSecurityCenterV2OrganizationNotificationConfig_basic(context map[string]interface{}) string {
+    return acctest.Nprintf(`
+resource "google_pubsub_topic" "scc_v2_organization_notification_config" {
+  name = "tf-test-topic-%s"
```

Comment:
I think you had these right before, but probably needs to be %{random_suffix}

Reply:
addressed